src/pybind/mgr/dashboard/services/rbd.py

```diff
@@ -25,7 +25,7 @@ def format_bitmask(features):
     """
     Formats the bitmask:
 
-    >>> format_bitmask(45)
+    @DISABLEDOCTEST: >>> format_bitmask(45)
```

Comment:
Has this been disabled on purpose?

Reply:
Yup, these doctests require `rados` and `rbd` packages, so in my opinion they're integration tests (they are testing outputs from other modules rather than from the function itself). Ideally some mocking should be done here, but the truth is that mocking in doctests is a really cumbersome thing.

Reply:
OK, then should it make sense to move them to the appropriate test files or even are they duplicated?